source/features/minimize-upload-bar.css

```diff
@@ -1,9 +1,10 @@
 /* `.js-upload-markdown-image` excludes non-comment attachment drop areas. See #2086 */
-.rgh-minimize-upload-bar .js-upload-markdown-image.is-default .drag-and-drop {
+/* `:not(.js-comment-update)` excludes comment edition forms, as they lack the upload image button. See #4542 */
+.rgh-minimize-upload-bar form:not(.js-comment-update) .js-upload-markdown-image.is-default .drag-and-drop {
 	display: none !important;
 }
 
-.rgh-minimize-upload-bar .js-upload-markdown-image.is-default textarea {
+.rgh-minimize-upload-bar form:not(.js-comment-update) .js-upload-markdown-image.is-default textarea {
```

Comment:
This is a GitHub bug by the way, if you'd like, can you report it? "Upload button not showing on mobile when editing comments"